buildspecs/release-javadoc.yml

```diff
@@ -22,6 +22,7 @@ phases:
     - RELEASE_VERSION=`mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec`
     -
     - aws s3 sync target/site/apidocs/ $DOC_PATH/$RELEASE_VERSION/
+    - aws s3 cp ./scripts/crosslink_redirect.html $DOC_PATH/$RELEASE_VERSION/ --acl="public-read"
```

Comment:
Do you mind applying --acl=public-read to the first sync as well, while you're in here?

Do we still need the cp for crosslink on line 29?

Reply:
Sure adding acl
Yeah that should be redundant now, removing